### PR TITLE
CloudFormation Resource Specification 18.6.0

### PR DESCRIPTION
**Pull request by kddejong**
fixes https://github.com/aws-cloudformation/cfn-python-lint/issues/1706

*Description of changes:*
- Update specs to 18.6.0

```
scripts/update_specs_from_pricing.py # requires Boto3 and Credentials
scripts/update_specs_services_from_ssm.py # requires Boto3 and Credentials
cfn-lint --update-specs # https://github.com/aws-cloudformation/cfn-python-lint/pull/1383\#issuecomment-629891506
cfn-lint --update-iam-policies
cfn-lint --update-documentation
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/cfnlint/data/CloudSpecs/ap-northeast-1.json [hunk 1]
```diff
@@ -56529,12 +56603,6 @@
           "Required": false,
           "UpdateType": "Immutable"
         },
-        "Name": {
-          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-component.html#cfn-imagebuilder-component-name",
-          "PrimitiveType": "String",
-          "Required": true,
-          "UpdateType": "Immutable"
-        },
```
Comment (PatMyron) [inline src/cfnlint/data/CloudSpecs/ap-northeast-1.json:56537]: lost the `Name` property from [5 ImageBuilder resource types](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_ImageBuilder.html) in this week's [resource specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html)

same reason as [`AWS::StepFunctions::Activity.Name`](https://github.com/aws-cloudformation/cfn-python-lint/pull/1715#discussion_r496397195)

[(incorrectly marking non-`readOnlyProperties` as `readOnlyProperties` for `GetAtt` support)](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/68#issuecomment-665243220)

File: src/cfnlint/data/CloudSpecs/us-east-1.json [hunk 2]
```diff
@@ -15828,7 +15864,7 @@
     "AWS::ECS::Service.NetworkConfiguration": {
       "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-networkconfiguration.html",
       "Properties": {
-        "AwsVpcConfiguration": {
+        "AwsvpcConfiguration": {
```
Comment (PatMyron) [inline src/cfnlint/data/CloudSpecs/us-east-1.json:15867]: fixes https://github.com/aws-cloudformation/cfn-python-lint/issues/1706

File: src/cfnlint/data/CloudSpecs/us-east-1.json [hunk 3]
```diff
@@ -70268,6 +70363,7 @@
         "EC2SpotFleetRequestAverageNetworkOut",
         "ECSServiceAverageCPUUtilization",
         "ECSServiceAverageMemoryUtilization",
+        "KafkaBrokerStorageUtilization",
```
Comment (PatMyron) [inline src/cfnlint/data/CloudSpecs/us-east-1.json:70366]: nice, https://github.com/aws-cloudformation/cfn-python-lint/pull/1682 working well 🥳

**Comment by PatMyron** [comment]
@kddejong seems to be missing https://github.com/aws-cloudformation/cfn-python-lint/pull/1725

File: src/cfnlint/data/CloudSpecs/ap-northeast-1.json (hunk 1 above)
Comment (kddejong) [reply src/cfnlint/data/CloudSpecs/ap-northeast-1.json:56537]: @PatMyron good eye.  Should we patch this in?

**Comment by PatMyron** [comment]
@kddejong should we start combining these PRs with the release PRs? bit weird when something gets merged in between those PRs if it's something that needs the maintenance script to actually take effect

File: src/cfnlint/data/CloudSpecs/ap-northeast-1.json (hunk 1 above)
Comment (kddejong) [reply src/cfnlint/data/CloudSpecs/ap-northeast-1.json]: Patched.

**Review by PatMyron — APPROVED** [review]

**Pull request merged**